src/pages/components/contained-list/usage.mdx

```diff
@@ -69,12 +69,34 @@ actions and interactive elements.
 
 ## Live demo
 
-<InlineNotification>
-
-**Coming soon:** The live demo for contained list is underdevelopment and will
-be available soon.
-
-</InlineNotification>
+<ComponentDemo
+  components={[
+    {
+      id: 'contained-list',
+      label: 'Contained list',
+    },
+  ]}
+>
+  <ComponentVariant
+    id="contained-list"
+    knobs={{
+      ContainedList: ['kind', 'size', 'isInset'],
+    }}
+    links={{
+      React:
+        'https://react.carbondesignsystem.com/?path=/story/components-containedlist--default',
+    }}
+  >
+    {`
+      <ContainedList label="List title" kind="on-page">
```

Comment:
could we not specify the kind here? looks like "on-page" is the default and the story is duplicating it when the kind knob is used:
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/40550942/223548991-1b3f28c9-4216-4f1e-8ee6-dd0a5511c30f.png">

```suggestion
      <ContainedList label="List title">
```